templates/tdarr_node.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>tdarr</Name>
```

Comment:
Probably worth renaming this to something exclusive

Reply:
Thought I'd changed that, good catch thanks. Updated.